README.md

```diff
@@ -1,141 +1,5 @@
-```
-data-models-packer -verifyMetadata data/test
-```
-
-Unpack an unencrypted package into a directory.
-```
-data-models-packer -out data/test test.tar.gz
-```
-
-Unpack an encrypted data archive (with the passphrase in a file).
-```
-data-models-packer -keyPath key.asc -keyPassPath pass.txt test.tar.gz.gpg
-```
-
-Unpack an encrypted data archive (with the passphrase in an env var).
-```
-PACKER_KEYPASS=foobar data-models-packer -keyPath key.asc test.tar.gz.gpg
-```
-
-### Arguments
-
-```
--comp string
-      The compression method to be used: '.zip', '.tar.gz', '.tar.gzip',
-      '.tar.bz2', or '.tar.bzip2'. If omitted, the '.tar.gz' method will be
-      used for packing and the file extension will be used to infer a method
-      for unpacking or the STDIN stream is assumed to be uncompressed.
--dataVersion string
-      The specific version of the data in the package.
--etl string
-      The URL of the ETL code used to generate data. Should be specific to the
-      version of code used and remain that way over time.
--keyPassPath string
-      The filepath to the file containing the passphrase needed to access the
-      private key. If omitted, the 'PACKER_KEYPASS' environment variable will
-      be used, if that is unset, the private key is assumed to be unprotected.
--keyPath string
-      The filepath to the public key to use for encrypting packaged data or to
-      the private key to use for unpacking encrypted data. If omitted, the data
-      is assumed to be unencrypted.
--model string
-      The data model to operate against.
--modelVersion string
-      The specific version of the model to operate against. Defaults to the
-      latest version of the model.
--out string
-      The directory or filename that should be written to. If omitted, data
-      will be unpacked into the current directory or packed onto STDOUT.
--service string
-      The URL of the data models service to use for fetching schema
-      information.
--site string
-      The site that generated the data.
--verifyOnly
-      Only verify an existing 'metadata.csv' file in the given data directory.
-      Do not package the directory.
-```
-
-## Known Bugs
-
-The combination of the tar and gzip Go libraries for writing appears to
-suffer from a bug which makes the  resulting file unreadable for the standard
-command line tools. This means that packages output by data model packer cannot
-be read by standard tar and gzip tools. Unfortunately, .tar.gz is the default
-format output by data model packer and the only compression method compatible
-with encryption at this time. Of course, they can be unpacked by data model
-packer.
-
-The Go zip library is unable to decompress zip archives that use the DEFLATE64
-compression method.
-
-## Developers
-
-Install the dependencies.
-```
-make install
-```
-
-Build the binary.
-```
-make build
-```
-
-Run tests.
-```
-make test
-```
+A Go library for handling compressed and optionally encrypted data packages.
```

Comment:
Maybe for people who never look at badges we could have a sentence like "Documentation is available [here](https://godoc.org/github.com/infomodels/datapackage)."